lib/perl/Genome/Config/Profile/Item.pm

```diff
@@ -155,7 +155,8 @@ sub _copy_file_to_allocation {
         $allocation->reallocate();
     }
     else {
-        my $config_file = 'model.' . $self->id . $filename . '.yaml';
+        my ($file_name, $file_dirs, $file_suffix) = File::Basename::fileparse($config_path, qr/\.[^.]+/);
```

Comment:
If you don't need the dirs and suffix you can omit them.  (This suggestion also corrects the variable name from the nonexistent `$config_path`.)
```suggestion
        my ($file_name) = File::Basename::fileparse($original_file_path, qr/\.[^.]+/);
```

Reply:
Thanks, forgot to edit my copy/paste!